server/app.js

```diff
@@ -1,11 +1,17 @@
 const express = require('express');
+const path = require('path');
 const app = express();
 const port = 2001;
 const db = require('../database-mongodb/index.js')
 
-app.use(express.static('./public'));
+app.use(express.static(path.join(__dirname, '../public')));
 app.use(express.json());
 app.use(express.urlencoded({extended: true}));
+app.use(function(req, res, next) {
```

Comment:
Im intrigued by whats going on here! I'm guessing you will list all the URLs in res.header  for the microservices your pulling from?

Reply:
For now I am, but it might change when we set everything up on AWS and/or set up env vars!

Reply:
oooohhhh. How exciting, nice work @krisdo !